src/vivarium_gates_nutrition_optimization_child/data/loader.py

```diff
@@ -814,12 +822,13 @@ def load_gbd_2021_exposure(key: str, location: str) -> pd.DataFrame:
         probabilities["sex"] = probabilities["sex"].str.capitalize()
         # get age start and end from age group ID
         age_bins = get_data(data_keys.POPULATION.AGE_BINS, location).reset_index()
+        age_bins["age_group_id"] = list(metadata.AGE_GROUP.GBD_2021)
```

Comment:
I would be careful about defining age group ID like this because it's very dependent on the ordering of age_bins data and our age group metadata. You can replace this line and the one above it with

`age_bins = utilities.get_gbd_age_bins()`

and update utilities.get_gbd_age_bins to be 

`def get_gbd_age_bins(age_group_ids: List[int] = None) -> pd.DataFrame:
    age_bins = gbd.get_age_bins().rename(columns={"age_group_years_start": "age_start", "age_group_years_end": "age_end"})
    return age_bins`

Reply:
sorry i can't get the code formatting to work properly

Reply:
Okay I think this makes sense and I updated the code